components/namespace-lister/base/deployment.yaml

```
@@ -40,10 +40,10 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 128Mi
+            memory: 256Mi
```

As long as we're increasing things, maybe let's set the memory limit to something much higher than what we're requesting?

```suggestion
            memory: 1Gi
```

---

makes sense. commited